src/docs/multiomics_submission.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Multi-omics dataset submission
+author:
+  - name: MGnify
```

Comment:
@Ales-ibt feel free to add yourself as an individual author too if you like.

Reply:
I can see all the documentation pages have MGnify or MGnify team as author. I'll stick to that :D